app/controllers/oaisys/pmh_controller.rb

```diff
@@ -2,25 +2,49 @@
 
 module Oaisys
   class PMHController < ApplicationController
+    layout 'oai_pmh_response.xml.builder'
```

Comment:
I wasn't able to get rails to use application.builder default. I moved the layout definition to application_controller because without it it errors, not sure why